test-data/unit/check-flags.test

```diff
@@ -509,3 +509,72 @@ x, y = 1, 2  # type: Unchecked, Unchecked
 [out]
 main:4: error: Type of variable becomes "Any" due to an unfollowed import
 main:6: error: A type on this line becomes "Any" due to an unfollowed import
+
+[case testDisallowAnyExprUnannotatedFunction]
+# flags: --disallow-any=expr
+def g(s):
+    return s  # E: Expressions of type "Any" are disallowed (has type "Any")
+
+g(0)
```

Comment:
It'd be nice to put a short comment on this line explaining what it's testing and why it's not an error.